prime-router/src/testIntegration/resources/datatests/mappinginventory/cx/cx-identifier-system.hl7

```diff
@@ -0,0 +1,2 @@
+MSH|^~\&#|||||20230501102531-0400||ORU^R01^ORU_R01|12345||2.5.1
+OBR|1|two|three|68991-9^Epidemiologically Important Information^LN^^^^^^^^^^^2.16.840.1.113883.6.1|||||||||||||||||||||unknown||||||||||||||||||||||||||||IDNumber^checkdigit^checkdigitscheme^http://hl7.org/fhir/sid/us-ssn&ISO&PLAC^PLAC^blah2&blah2&CLIP^19880704^20880704^AJCode&Assigning Jurisdiction&AJSys^AACode&Assigning Agency&AASys
```

Comment:
This is still triggering the assigner/organization case instead of the system/uri case. I think CX.4.3 has to be ISO to exercise that case, but it's CX.4.2 instead.